docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -321,26 +317,37 @@ For more information, see xref:../troubleshooting/troubleshooting-driver-plugins
 
 . Verify that you pair the virtual switch with a network card (wired or wireless) that is connected to the network.
 
-. Use the configuration option `hyperv-virtual-switch` or startup flag `--hyperv-virtual-switch` to set the name of the external virtual switch you want to use for {project}.
+. Set the environment variable `HYPERV_VIRTUAL_SWITCH` to the name of the external virtual switch you want to use for {project}.
```

Comment:
I don't recall seeing these changes before -- we need the original wording here. Using the `HYPERV_VIRTUAL_SWITCH` environment variable isn't generally recommended; the related configuration option is more appropriate.

This comment applies down to line 350, where the "Next Steps" heading is lowered.

Reply:
Hey, thanks for catching that. I guess I accidentally branched from an older local master branch. I will fix this soon.

Reply:
Glad I caught it. :)

Thank you for working through all of the change requests!

Reply:
@rhnvrm can you fix this and resent so we can merge it.

Reply:
@robin-owen @praveenkumar, fixed this